src/clients/admin/types.ts

```diff
@@ -79,6 +79,16 @@ export interface DeleteGroupsOptions {
   groups: string[]
 }
 
+interface MemberRemoval {
+  memberId: string
+  reason: string
+}
+
+export interface RemoveMembersFromConsumerGroupOptions {
+  groupId: string
+  members?: Nullable<(string | MemberRemoval)[]>
```

Comment:
Please change this. I would only accept MemberRemoval.
Also, the type of Member should be:

```
interface MemberRemoval {
  memberId: string
  reason?: string
}
```